sidebars.js

```diff
@@ -120,7 +119,6 @@ module.exports = {
   },
   {
     "Okteto Self-Hosted": [
-      "self-hosted/overview-self-hosted",
```

Comment:
CAn you remove the overview.mdx since we are not displaying it everywhere, and for now redirect to get-started/overview?